### PR TITLE
docs:update quick-start document

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes partially #461 

Reference to #

**Special notes for your reviewer**:

/cc @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -1,121 +1,194 @@
...
+            "host": "'$DOCKER_HOST'"
+        }
+ }' ''$HOST_URL'/api/v1/clouds'
+ ```
+
+ The script above would create a cloud at your local computer, then you could set up the project.
+
+ ## Create a project
+
+ A project represents a group to manage a set of related pipelines. project must has a credential to access your VCS repository, and now Cyclone supports three types of VCS: GitHub, GitLab and SVN.
```
[the remark this answers is not in the document]
Comment (supereagle) [reply docs/quick-start.md]: Replace VCS with SCM

File: docs/quick-start.md [hunk 2]
```diff
@@ -1,121 +1,205 @@
...
+            "host": "'$DOCKER_HOST'"
+        }
+ }' ''$HOST_URL'/api/v1/clouds'
+ ```
+
+ The command above would create a cloud at your local computer, then you could set up the project.
+
+ ## Create a project
+
+ A project represents a group to manage a set of related pipelines. project must have a credential to access your SCM(Software Configuration Management) repository, and now Cyclone supports three types of SCM: GitHub, GitLab and SVN.
```
Comment (supereagle) [inline docs/quick-start.md]: SCM: Source code management

File: docs/quick-start.md [hunk 3]
```diff
@@ -1,121 +1,205 @@
...
+ curl -X POST -H "Content-Type: application/json" -d '{
+        "name": "myTestCloud",
+        "type": "Docker",
+        "docker": {
+            "host": "'$DOCKER_HOST'"
+        }
+ }' ''$HOST_URL'/api/v1/clouds'
+ ```
+
+ The command above would create a cloud at your local computer, then you could set up the project.
```
Comment (supereagle) [inline docs/quick-start.md]: The command above would registry your local computer as a cloud.

File: docs/quick-start.md (hunk 2 above)
Comment (zhujian7) [reply docs/quick-start.md]: wow... embarrassed, i will fix it.

**Pull request merged**